src/Custom-sample.h

```diff
@@ -362,9 +362,7 @@ static const char DATA_ESPEASY_DEFAULT_MIN_CSS[] PROGMEM = {
 // #define USES_P100   // DS2423 counter
 // #define USES_P101   // WakeOnLan
 // #define USES_P102   // PZEM004Tv3
-// #define USES_P103   // Atlas_EZO_pH
-// #define USES_P104   // Atlas_EZO_EC
-// #define USES_P105   // Atlas_EZO_ORP
+#define USES_P103   // Atlas_EZO_pH_ORP_EC
```

Comment:
In the sample file all plugins should be put in comments

Reply:
Oh sorry, I changed it